src/transformers/models/layoutxlm/tokenization_layoutxlm.py

```diff
@@ -44,6 +42,113 @@
 logger = logging.get_logger(__name__)
 
 
+LAYOUTXLM_ENCODE_KWARGS_DOCSTRING = r"""
```

Comment:
Thanks for the improvement! Was definitely needed.

I'd recommend here actually to just create a single  `LAYOUTXLM_KWARGS_DOCSTRING` instead of having it split up between an kwargs and additional_kwargs docstrings, since all methods (like encode_plus, batch_encode_plus, prepare_for_model and call) all have the same kwargs.

Reply:
Good point.